src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -0,0 +1,720 @@
+
+            jdl.append("Queue 1\n")
+
+        return jdl
+
+    def customizePerJob(self, job):
+        """
+        JDL additions just for this implementation. Over-ridden in sub-classes
+        These are the Glide-in specific bits
+        """
+        jdl = []
+        jobCE = job['location']
+        if not jobCE:
+            # Then we ended up with a site that doesn't exist?
+            logging.error("Job for non-existant site %s", job['location'])
+            return jdl
+
+        if self.submitWMSMode and len(job.get('possibleSites', [])) > 0:
+            strg = ','.join(map(str, job.get('possibleSites')))
+            jdl.append('+DESIRED_Sites = \"%s\"\n' % strg)
+        else:
+            jdl.append('+DESIRED_Sites = \"%s\"\n' % (jobCE))
+
+        if self.submitWMSMode and len(job.get('potentialSites', [])) > 0:
+            strg = ','.join(map(str, job.get('potentialSites')))
+            jdl.append('+ExtDESIRED_Sites = \"%s\"\n' % strg)
+        else:
+            jdl.append('+ExtDESIRED_Sites = \"%s\"\n' % (jobCE))
+
+        jdl.append('+WMAgent_RequestName = "%s"\n' % job['requestName'])
+        match = GROUP_NAME_RE.match(job['requestName'])
+        if match:
+            jdl.append('+CMSGroups = %s\n' % classad.quote(match.groups()[0]))
+        else:
+            jdl.append('+CMSGroups = undefined\n')
+
+        jdl.append('+WMAgent_SubTaskName = "%s"\n' % job['taskName'])
+        jdl.append('+CMS_JobType = "%s"\n' % job['taskType'])
+
+        # Handling for AWS, cloud and opportunistic resources
+        jdl.append('+AllowOpportunistic = %s\n' % job.get('allowOpportunistic', False))
+
+        # dataset info
+        if job.get('inputDataset'):
+            jdl.append('+DESIRED_CMSDataset = "%s"\n' % job['inputDataset'])
+        else:
+            jdl.append('+DESIRED_CMSDataset = undefined\n')
+        if job.get('inputDatasetLocations'):
+            jdl.append('+DESIRED_CMSDataLocations = "%s"\n' % ','.join(job['inputDatasetLocations']))
+        else:
+            jdl.append('+DESIRED_CMSDataLocations = undefined\n')
+
+        # HighIO jobs
+        jdl.append('+Requestioslots = %d\n' % job.get('highIOjob', 0))
+
+        # Performance and resource estimates
+        numberOfCores = job.get('numberOfCores', 1)
+        requestMemory = int(job['estimatedMemoryUsage']) if job.get('estimatedMemoryUsage', None) else 1000
+        requestDisk = int(job['estimatedDiskUsage']) if job.get('estimatedDiskUsage', None) else 20*1000*1000*numberOfCores
+        maxWallTimeMins = int(job['estimatedJobTime'])/60.0 if job.get('estimatedJobTime', None) else 12*60
+        jdl.append('request_memory = %d\n' % requestMemory)
+        jdl.append('request_disk = %d\n' % requestDisk)
+        jdl.append('+MaxWallTimeMins = %d\n' % maxWallTimeMins)
+
+        # How many cores job is using
+        jdl.append('machine_count = 1\n')
```

Comment:
This would be something you'd only set for the HTCondor parallel universe (which we definitely aren't using).  Let's remove it.